libpod/networking_linux.go

```diff
@@ -139,10 +139,15 @@ func (r *Runtime) setupRootlessNetNS(ctr *Container) (err error) {
 	defer ctr.rootlessSlirpSyncR.Close()
 	defer ctr.rootlessSlirpSyncW.Close()
 
-	path, err := exec.LookPath("slirp4netns")
-	if err != nil {
-		logrus.Errorf("could not find slirp4netns, the network namespace won't be configured: %v", err)
-		return nil
+	path := r.config.NetworkCmdPath
+
+	if path == "" {
+		var err error
+		path, err = exec.LookPath("slirp4netns")
+		if err != nil {
+			logrus.Errorf("could not find slirp4netns, the network namespace won't be configured: %v", err)
```

Comment:
Maybe I misunderstood the suggestion from @rhatdan, but thought 'slirp4netns` would no longer be hardcoded, but would be part of the passed in value?

Reply:
for configuring a slirp network we need exactly `slirp4netns`